src/wechaty/user/room.py

```diff
@@ -289,14 +297,176 @@ def is_ready(self) -> bool:
 
         await self.puppet.room_add(self.room_id, contact.contact_id)
 
-    async def quit(self) -> None:
+    async def delete(self, contact: Contact):
+        """
+        delete room
+        """
+        log.info('Room delete<%s>', contact)
```

Comment:
As my understanding, our `log` is coming from:

```py
log = logging.getLogger(__name__)
```

I'd like to suggest that we get the log for the current module name, like:

```py
log = logging.getLogger('Room')
```

and then we can use this log message format:

```py
log.info('delete(%s)', contact)
```
Which will be perfect align with our TS Wechaty logging style.

Please feel free to let me know what you think, thank you very much.

Reply:
Yes, this way can simplify message format.